docs/book/src/plugins/creating-plugins.md

```diff
@@ -0,0 +1,78 @@
+# Creating your own plugins
+
+## Overview
+
+You can extend KubeBuilder API to create your plugins. If you are using the [its extension][extending-cli] then, it means that your plugin will be implemented in your project and register then to the CLI as it has been done by the [SDK][sdk] project. See its [cli code][sdk-cli-pkg] as an example.
+
+## Language-based Plugins
+
+KubeBuilder offers the Golang-based operator plugins, which will help its CLI tool users create projects following the [Operator Pattern][operator-pattern] Golang language.
+
+Now see that [SDK][sdk], for example, has its language plugins for [Ansible][sdk-ansible] and [Helm][sdk-helm], which are similar options but for users who would like to work with these respective languages and stacks instead of Golang.
```

Comment:
```suggestion
The [SDK][sdk] project, for example, has language plugins for [Ansible][sdk-ansible] and [Helm][sdk-helm], which are similar options but for users who would like to work with these respective languages and stacks instead of Golang.
```